tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/builder/IncrementalBuilderCancellationTest.xtend

```diff
@@ -28,13 +33,14 @@ import static org.junit.Assert.*
  */
 class IncrementalBuilderCancellationTest extends AbstractIncrementalBuilderTest {
 
+	/* Cancel after n resources */
 	private static final AtomicInteger cancelCounter = new AtomicInteger(0);
```

Comment:
Proposal: "Iff greater zero, then cancel while building the n-th resource."

Reply:
:+1: